src/components/PracticeSection/MultipleChoice.js

```diff
@@ -542,11 +542,8 @@ export default function MultipleChoice() {
                               <Form.Check
                                 type="radio"
                                 value={option}
-                                label={
-                                  <div
-                                    dangerouslySetInnerHTML={{ __html: option }}
-                                  />
-                                }
+                                id={option}
```

Comment:
Was it okay to change this? Or was there a reason for the dangerous html

Reply:
Yes, lets close this and continue with the other